documentation/feature/parachute.md

```diff
@@ -14,11 +14,11 @@ Removes the altitude and descend speed UI elements when free-falling and parachu
 ### 1.2 Non-steerable parachute
 Adds a non-steerable parachute variant for jet pilots.
 
-### 1.3 Landing animation
-Smoothens the parachute landing animation.
+### 1.3 Parachute cutting and reserve parachutes
+You are now able to cut parachutes and deploy a reserve one.
 
-### 1.4 Reserve parachute
```

Comment:
you commented on the old diff :D the new one indicates that you can deploy the reserve parachute after cutting the primary.